rllib/env/policy_client.py

```diff
@@ -373,26 +373,25 @@ def _create_embedded_rollout_worker(kwargs, send_fn):
     # Since the server acts as an input datasource, we have to reset the
     # input config to the default, which runs env rollouts.
     kwargs = kwargs.copy()
-    del kwargs["input_creator"]
-
-    # Since the server also acts as an output writer, we might have to reset
-    # the output config to the default, i.e. "output": None, otherwise a
-    # local rollout worker might write to an unknown output directory
-    del kwargs["output_creator"]
+    kwargs["config"] = kwargs["config"].copy(copy_frozen=False)
+    config = kwargs["config"]
+    config.output = None
+    config.input_ = "sampler"
```

Comment:
lgtm. Was thinking that users might need access to other input types but I guess why would anyone use policy_client if they weren't doing environment sampling

Reply:
Yeah, that, too. But also note this is only for the extra RolloutWorker that we create on either the client (inference-mode=local) or on the server (inside the PolicyServerInput object!) for inference mode=remote.
The whole design is completely flawed imo, but we'll have to fix this separately, it's beyond the scope of this quick fix PR. This PR does NOT touch the original (bad) design.